url7a69e77641_benmouh_MySgbd_tgz-pJ8-Demo_CheckRowNumJ8/src/org/sgbd/Mysgbd/parser/MysgbdParserTokenManager.java

```diff
@@ -0,0 +1,1654 @@
+         break;
+      case 62:
+         if ((active1 & 0x100000000L) != 0L)
+            return jjStopAtPos(1, 96);
+         break;
+      case 65:
+      case 97:
+         return jjMoveStringLiteralDfa2_0(active0, 0x110080100000L, active1, 0xe2L);
+      case 69:
+      case 101:
+         return jjMoveStringLiteralDfa2_0(active0, 0x1980000000600800L, active1, 0L);
+      case 70:
+      case 102:
+         if ((active0 & 0x2000000000000L) != 0L)
+            return jjStartNfaWithStates_0(1, 49, 48);
+         break;
+      case 72:
+      case 104:
+         return jjMoveStringLiteralDfa2_0(active0, 0x2000000000008000L, active1, 0x100L);
+      case 73:
+      case 105:
+         return jjMoveStringLiteralDfa2_0(active0, 0x64000801000L, active1, 0x200L);
+      case 76:
+      case 108:
+         return jjMoveStringLiteralDfa2_0(active0, 0x8000020L, active1, 0L);
+      case 77:
+      case 109:
+         return jjMoveStringLiteralDfa2_0(active0, 0x4000000000000000L, active1, 0L);
+      case 78:
+      case 110:
+         if ((active0 & 0x100000000L) != 0L)
+         {
+            jjmatchedKind = 32;
+            jjmatchedPos = 1;
+         }
+         return jjMoveStringLiteralDfa2_0(active0, 0x4001e000000c0L, active1, 0x8L);
+      case 79:
+      case 111:
+         return jjMoveStringLiteralDfa2_0(active0, 0x6006880100f2000L, active1, 0x400L);
+      case 80:
+      case 112:
+         return jjMoveStringLiteralDfa2_0(active0, 0L, active1, 0x10L);
+      case 82:
+      case 114:
+         if ((active0 & 0x8000000000000L) != 0L)
+         {
+            jjmatchedKind = 51;
+            jjmatchedPos = 1;
+         }
+         return jjMoveStringLiteralDfa2_0(active0, 0x30000060000000L, active1, 0x804L);
+      case 83:
+      case 115:
+         if ((active0 & 0x100L) != 0L)
+         {
+            jjmatchedKind = 8;
+            jjmatchedPos = 1;
+         }
+         else if ((active0 & 0x2000000000L) != 0L)
+            return jjStartNfaWithStates_0(1, 37, 48);
+         return jjMoveStringLiteralDfa2_0(active0, 0x200L, active1, 0L);
+      case 84:
+      case 116:
+         return jjMoveStringLiteralDfa2_0(active0, 0x8000000000000000L, active1, 0L);
+      case 85:
+      case 117:
+         return jjMoveStringLiteralDfa2_0(active0, 0x41800000000000L, active1, 0x1L);
+      case 86:
+      case 118:
+         return jjMoveStringLiteralDfa2_0(active0, 0x400L, active1, 0L);
+      case 88:
+      case 120:
+         return jjMoveStringLiteralDfa2_0(active0, 0x7000000L, active1, 0L);
+      case 89:
+      case 121:
+         if ((active0 & 0x4000L) != 0L)
+            return jjStartNfaWithStates_0(1, 14, 48);
+         break;
+      case 124:
+         if ((active1 & 0x40000000000L) != 0L)
+            return jjStopAtPos(1, 106);
+         break;
+      default :
+         break;
+   }
+   return jjStartNfa_0(0, active0, active1);
+}
+static private int jjMoveStringLiteralDfa2_0(long old0, long active0, long old1, long active1)
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This method contains 96 lines of code, not including blank lines or lines with only comments, Java punctuation characters, identifiers, or literals. By comparison, 99% of the methods in the CodeGuru reference dataset contain fewer lines of code. Large methods might be difficult to read and have logic that is hard to understand and test. We recommend that you simplify this method or break it into multiple methods.